src/components/web_lighthouse_gauge/web_lighthouse_gauge.ts

```diff
@@ -0,0 +1,50 @@
+// Import the LitElement base class and html helper function
+import {html, property, customElement, PropertyValues} from 'lit-element';
+import {BaseElement} from '../base_element/base_element';
+import './web_lighthouse_gauge.scss';
+
+/**
+ * Lighthouse gauge element.
+ */
+@customElement('web-lighthouse-gauge')
+export class WebProgressBar extends BaseElement {
+  @property({type: Number}) score = 1;
+
+  // eslint-disable-next-line require-jsdoc
+  render() {
+    const score = this.score;
+
+    this.setAttribute('role', 'progressbar');
+    this.setAttribute('aria-valuemin', '0');
+    this.setAttribute('aria-valuemax', '100');
+    this.setAttribute('aria-valuenow', `${score * 100}`);
```

Comment:
I think this is the only value that would need to be updated on render.

Reply:
Ack